syntax = "proto3";

package fintekkers.models.transaction;

option java_multiple_files = true;

option java_outer_classname = "TransactionTypeProtos";

enum TransactionTypeProto {
  UNKNOWN=0;
  BUY=1;
  SELL=2;
  DEPOSIT=3;
  WITHDRAWAL=4;
  MATURATION=5;
  MATURATION_OFFSET=6;
}
